src/data/faq.json

```diff
@@ -87,6 +87,117 @@
+                        "active": true,
+                        "textblock": [
+                            "Warnings based on a rapid test result are not shared with other EU countries via the EFGS (European Federation Gateway Service), including Switzerland, as the EFGS terms of use do not provide for rapid test warnings to be shared."
+                        ]
+                    },
+                    {
+                        "title": "What is the difference between PCR tests, rapid antigen tests and self-tests?",
+                        "anchor": "rat_differences_pcr_rat",
+                        "active": true,
+                        "textblock": [
+                            "PCR tests are the 'gold standard' among corona tests. Sampling is performed by medical personnel - evaluation by laboratories.",
+                            "Rapid antigen tests: Have their name because the result is available quickly. They can only be performed by trained personnel. Similar to the PCR test, a nasal or throat swab is taken. However, in contrast to the PCR test, the evaluation takes place directly on site. Since March 8, 2021, everyone is entitled to at least one rapid test per week.",
+                            "Self-tests: Have their name because anyone can do these tests themselves, for example at home. The self-tests are intended for use by private individuals. For this purpose, sample collection and evaluation must be appropriately simple. The test can be performed with a nasal swab or saliva, for example. The Federal Institute for Drugs and Medical Devices, together with the Paul Ehrlich Institute, checks the quality and significance of the tests.",
+                            "Rapid and self-tests have a higher error rate compared to PCR tests. Therefore, after each positive rapid and self-test, a PCR test should always be done for confirmation."
+                        ]
+                    },
+                    {
+                        "title": "Can I save/manage multiple rapid antigen tests?",
+                        "anchor": "rat_multiple_rat",
+                        "active": true,
+                        "textblock": [
+                            "No, only one rapid test can be registered in the app at a time. A negative rapid test is displayed in the app for 48 hours. After this time you can register a new test. You also have the option to remove the negative rapid test and result, and then register a new rapid test. Positive rapid tests are no longer displayed in the app once they have been shared.",
+                            "PCR tests are handled independently from rapid tests. This means you can have one PCR test and one rapid test registered in the app at the same time."
+                        ]
+                    },
+                    {
+                        "title": "Do the rapid antigen tests have different risk calculation parameters than a PCR tests?",

```

Comment:
Wrong heading here

Reply:
haha, ok, here is the text of the TAN article. Alright, lets keep here title, but change text. should be then the same in DE.

Reply:
should be fixed now. thanks again for the hint, great help!